brunoapimentel/helloworldjava
# There, I fixed it.

This PR removes lots of unnecessary over-engineering added to the project just because cool Java kids are doing it, but having no effect in the outcome. 
From an `E N T E R P R I S E` POV, going that route will add development time and we have to deliver things.

Test steps:
- run it before changes
- run it after changes
- verify that the code is cleaner
- verify that the output is the same
